jdi-light-mobile-examples/src/main/java/io/github/com/custom/Result.java

```diff
@@ -0,0 +1,7 @@
+package io.github.com.custom;
+
+import com.epam.jdi.tools.DataClass;
+
+public class Result extends DataClass<Result> {
+    public String name, description, link;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)